etc/skydive.yml.default

```diff
@@ -104,6 +104,8 @@ agent:
     # stats_update: 1
   metadata:
     info: This is compute node
+  tcp_len_by_seq: true
```

Comment:
Ok I understand your point
But if you don't take in account overlapping, retransmission, reordering packets, you flow tcp size metric would not be correct.

So let's do this patch step by step, and just focus on the syn/fin/rst flag and ttl only, this part sound good for me except few exception.
Could you remove the "unrelated" parts to tcp len calculation, I will prefer to see that in a separate patch

It's a pity that we have a big timezone difference, I will try catch you on IRC tonight (morning for you)